userdocs/profiles.dev/docs/author-docs/dependencies.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 9
+---
+
+# Dependencies between artifacts
+
+It's possible to add dependencies between artifacts. Consider the following scenario:
+
+There is an artifact (`artifact B`) which needs a certain set of CRDs to be present on the cluster before it can be installed.
+In that case, `artifact B` can depend on `artifact A` which installs the necessary CRDs before `artifact B`.
+
+This is achieved by adding a `dependsOn` setting on the desired artifact's configuration.
+
+Such as:
+
+```yaml
+spec:
+  description: Depending artifacts
+  artifacts:
+    - name: artifact-a
+      kustomize:
+        path: crds/rbacs
+    - name: artifact-b
+      chart:
+        url: https://charts.bitnami.com/bitnami
+        name: nginx
+        version: "9.3.0"
+        defaultValues: |
+          service:
+            type: ClusterIP
+      dependsOn:
+        - name: artifact-a
+        # ... the name of any other further dependencies this artifact might have 
+```
+
+:::info
+At the time of this writing, nested-profile dependencies are not functional yet.
+:::
+
+_Note_: this uses flux's [dependsOn](https://fluxcd.io/docs/components/kustomize/kustomization/#kustomization-dependencies) mechanism in the background to make sure artifacts
```

Comment:
The rest of this page is a little implementation-y, very interesting to be sure, but is this the right place for it?

Reply:
Huh, maybe not. I'm not sure. I can remove it I guess. It isn't really relevant.

Reply:
I think it would definitely be a good thing to have in the developer docs, so don't lose it

Reply:
Wait, only the note, or this whole thing? :D

Reply:
the whole thing from there down